### PR TITLE
pls cancel 2020-10-15 and restart with this

### PR DESCRIPTION
**Pull request by vanhauser-thc**
Sadly I introduced a big bug in the 2020-10-15 experiment. please cancel it as it is pointless.
this PR fixes the run ... sorry!

### CODE REVIEW

**Comment by inferno-chromium** [comment]
experiment stopped, will wait for ci before merge.

**Comment by vanhauser-thc** [comment]
thanks!

File: service/experiment-requests.yaml [hunk 1]
```diff
@@ -18,6 +18,14 @@
...
 # Please add new experiment requests towards the top of this file.
 
+- experiment: 2020-10-16
+  fuzzers:
+    - aflplusplus
+    - aflplusplus_optimal
+    - aflplusplus_memcache_2000mb
+    - aflplusplus_memcache_2mb
+    - aflplusplus_memcache_pre
+
```
Comment (jonathanmetzman) [inline service/experiment-requests.yaml:28]: Could you please add:
```
description: "<YOUR DESCRIPTION HERE>"
```
I want to test our new feature.

Comment (vanhauser-thc) [reply service/experiment-requests.yaml:29]: done, and presubmit is happy

**Comment by vanhauser-thc** [comment]
I think this can be merged now. Optimal uses the same commit id

**Comment by vanhauser-thc** [comment]
honggfuzz canceled? weird.

**Pull request merged**